modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -61,6 +63,10 @@ public Page<Object> getRelatedObjectEntriesPage(
 			String objectRelationshipName, Pagination pagination)
 		throws Exception {
 
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153324"))) {
+			throw new NotFoundException("HTTP 404 Not Found");
```

Comment:
I would not add the message, just like https://github.com/liferay/liferay-portal/blob/7a374aef330454753c594a198d5ee2f6247b02d7/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java#L430. 

There's no other place that passes that message.

Reply:
Done!